lib/Agrammon/OutputFormatter/Excel.pm6

```diff
@@ -87,6 +90,7 @@ sub input-output-as-excel(
 
     my $col = 0;
     my $last-module = '';
+    my $last-order = -1;
```

Comment:
This particular last-order declaration I see being declared here and assigned to (`$last-order = %rec<order>;`) below, but I don't see it being used anywhere?

Reply:
Already removed in the last commit which probably came after your review.